src/main/resources/default/taglib/w/multiSelect.html.pasta

```diff
@@ -35,7 +36,7 @@
         multiSelect({
             id: '@id',
             name: '@name',
-            noMatchesText : '@i18n("Select2.noMatches")',
+            noMatchesText : '@i18n("template.html.select2.noMatches")',
```

Comment:
would it make sense to make these keys more generic and drop references to select2?

Reply:
i think fabian is already working on this :)

Reply:
+1